lib/tasks_private/instance_types.rake

```diff
@@ -0,0 +1,83 @@
+namespace 'aws:extract' do
```

Comment:
extract might not be the best name for the namespace here but I wanted to have e.g. `rake aws:extract:regions` and `rake aws:extract:instance_types`

Reply:
This is fine.